scripts/create-public-collection-dumps.sql

```diff
@@ -0,0 +1,36 @@
+BEGIN TRANSACTION;
+
+-- duplicate user_collection table with public collections only
+
+CREATE table if not exists public_user_collection (LIKE bookbrainz.user_collection INCLUDING ALL);
```

Comment:
I would prefer to have these tables named starting with tmp_  to make it clear that they are dump tables. The clean-public-collection-dump-tables.sql script is a bit terrifying to read without the knowledge that these are not temp tables.